src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -91,6 +105,16 @@ public void setId(String id) {
         this.id = id;
     }
 
+    private List<EnhancedAlert> enhancedAlerts;
```

Comment:
all other member variables are declared at the top of the class.

Reply:
Fixed